client/web/compose/src/components/PageBlocks/Navigation/NavTypes/ComposePage.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <tr>
+    <td />
+    <td>
+      <b-form-group :label="$t('navigation.fieldLabel')">
+        <b-form-input
+          v-model="value.options.item.label"
+          type="text"
+          size="sm"
+        />
+      </b-form-group>
+    </td>
+    <td>
+      <b-form-group :label="$t('navigation.composePage')">
+        <vue-select
+          key="pageID"
+          v-model="value.options.item.pageID"
+          :placeholder="$t('navigation.none')"
+          :options="tree"
+          class="bg-white"
+          label="title"
+          :reduce="f => f.pageID"
+          option-value="pageID"
+          option-text="title"
+          @input="updateLabelValue"
+        />
+      </b-form-group>
+    </td>
+    <td>
+      <b-form-group :label="$t('navigation.openIn')">
+        <b-form-select
+          v-model="value.options.item.target"
+          :options="openInType"
+          size="sm"
+        />
+      </b-form-group>
+    </td>
+    <td
+      v-if="selectedPageChildren(value.options.item.pageID)"
+      cols="12"
+      sm="6"
+      class="align-middle text-center"
+    >
+      <b-form-group class="m-0">
+        <b-form-checkbox
+          v-model="value.options.item.displaySubPages"
+          name="enabled-button"
```

Comment:
No need for names (remove from others aswell), also do we really need text-left class (we don't just remove text-center)?